cmd/checkapi/config.yaml

```diff
@@ -0,0 +1,29 @@
+ignored_paths:
```

Comment:
I assume these are being ignored because we want checkapi to pass, but there are pre-existing issues? Are there issues we can open for the remaining failures so we don't forget about this, or at least notify code owners?

Reply:
yes, there are issues. For at least some of them, I have issues open and they are present in comments inline.